package/lib/src/beamer_delegate.dart

```diff
@@ -446,7 +436,7 @@ class BeamerDelegate extends RouterDelegate<RouteInformation>
 
   /// {@template beamToNamed}
   /// Beams to [BeamLocation] that has [uri] contained within its
-  /// [BeamLocation.pathBlueprintSegments].
+  /// [BeamLocation.pathPatterns] segments.
```

Comment:
It seems that mentioning `pathBlueprintSegments` was a bit misleading here. I would rewrite this sentence completely;

> Configures and beams to a [BeamLocation] that supports `uri` within its [BeamLocation.pathPatterns].